packages/protocol/contracts/governance/Validators.sol

```diff
@@ -402,6 +402,8 @@ contract Validators is
     require(uptime <= FixidityLib.fixed1().unwrap(), "Uptime cannot be larger than one");
     uint256 numerator;
     uint256 denominator;
+    uint256 grace_period=6250000000000000000000;
+    uptime = uptime + grace_period > FixidityLib.fixed1().unwrap() ? FixidityLib.fixed1().unwrap() : uptime + grace_period;
```

Comment:
I'm not sure this ternary even evaluates as intended.
Is it `(uptime + grace_period) > fixed1` or `uptime + (grace_period > fixed1)`
I assume the former
```suggestion
    uptime = Math.min(uptime + grace_period, FixidityLib.fixed1().unwrap());
```

Reply:
It is correct, but definitely `Math.min()` is preferable.